app/dashboard/dashboard.tsx

```diff
@@ -51,7 +50,7 @@ const StyledFallback = styled(Avatar.Fallback, {
 
 type DashboardProps = {
   projects?: Array<{ id: string; title: string }>;
-  user?: User;
+  user: User;
```

Comment:
if that's true, no need for optional checking for user in all accessors?

Reply:
hm?

Reply:
down the line a lot of defensive checking user?.something, which make it look like user is optional

Reply:
I think I looked at some wrong diff